onnxruntime/test/fuzzing/include/OnnxPrediction.h

```diff
@@ -46,17 +46,20 @@ class OnnxPrediction
   //
   OnnxPrediction(onnx::ModelProto& onnx_model);
 
-  // Deletes the prediction object 
+  // The following constructor is meant for initializing using flatbuffer model.
+  // Memory buffer pointing to the model
+  //
+  OnnxPrediction(const std::vector<char>& modelData);
+
```

Comment:
nit: model_data

A lot of the variable and function naming in this file does not conform to the coding standards, so perhaps that can be fixed in a separate PR.

Reply:
Are we following the camel naming convention for the variables. In this file I found both camel convention and "_" separate variables. I should read the coding style document. Is the on the group wiki?

Reply:
fixed the variable anmes